src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -79,8 +99,21 @@ public static int Main(string[] args)
             app.Description = "Initializes empty project for .NET Platform";
             app.HelpOption("-h|--help");
 
+            var csharp = app.Option("-c#|--csharp", "Create an C# project", CommandOptionType.NoValue);
```

Comment:
I think that explicit form should be either (`dotnet new --lang csharp`, `dotnet new --lang fsharp`) or (`dotnet new c#`, `dotnet new csharp`, ...)

I'm also wondering to maybe provide the source file name itself and if someone provide extension .cs then use csharp template and if .fs then use fsharp template. It would be something like `dotnet new Program.cs`. cc @piotrpMSFT @davidfowl @blackdwarf @anurse @brthor @Sridhar-MS